tasks/core/config.yml

```diff
@@ -37,18 +43,13 @@
       register: result
       changed_when: result is success
       notify: nextcloud update htaccess
+      failed_when: (not (result.stdout | search('successfully imported'))
```

Comment:
`search` is a test, not a filter, and should be called with `result.stdout is not search...`